nuxeo-drive-core/src/main/java/org/nuxeo/drive/adapter/impl/AbstractDocumentBackedFileSystemItem.java

```diff
@@ -293,6 +289,17 @@ public FileSystemItem move(FolderItem dest) {
         }
     }
 
+    /*--------------------- Object -----------------*/
+    @Override
+    public boolean equals(Object obj) {
+        return super.equals(obj);
```

Comment:
I don't see the point of this. If it's to shut up some Sonar warning then we should deal with Sonar but not pollute our code.

Reply:
It's to comply with the following Sonar rule:
> Extend a class that overrides equals and add fields without overriding equals in the subclass, and you run the risk of non-equivalent instances of your subclass being seen as equal, because only the superclass fields will be considered in the equality test.

The goal is not to shut up the rule which I find relevant: I want to explicitely show that the `equals` implementation has not been forgotten and relies on the parent class even though new members have been added to the child class.